job.py

```diff
@@ -174,16 +174,16 @@ def runReform(target_dir, ref_fasta, ref_gff, timestamp, position, chrom, in_fas
 
 
 def send_email(email, timestamp):
-    # calculate 72h DDL
-    deadline = datetime.now() + timedelta(hours=72)
+    # calculate 168h DDL
+    deadline = datetime.now() + timedelta(hours=168)
     deadline_str = deadline.strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Let's be a little more human readable. Yes, the deadline is exactly 168 hours, but let's not put the seconds or minutes. Simply say the date.